sklearn/tests/test_kernel_approximation.py

```diff
@@ -95,16 +95,16 @@ def test_additive_chi2_sampler():
     kernel = large_kernel.sum(axis=2)
 
     # approximate kernel mapping
-    transform = AdditiveChi2Sampler(sample_steps=3)
-    X_trans = transform.fit_transform(X)
-    Y_trans = transform.transform(Y)
+    transformer = AdditiveChi2Sampler(sample_steps=3)
+    X_trans = transformer.fit_transform(X)
+    Y_trans = transformer.transform(Y)
 
     kernel_approx = np.dot(X_trans, Y_trans.T)
 
     assert_array_almost_equal(kernel, kernel_approx, 1)
 
-    X_sp_trans = transform.fit_transform(csr_matrix(X))
-    Y_sp_trans = transform.transform(csr_matrix(Y))
+    X_sp_trans = transformer.fit_transform(csr_matrix(X))
+    Y_sp_trans = transformer.transform(csr_matrix(Y))
```

Comment:
This renaming is relevant, yet it is unrelated to the scope of this PR. Hence I am -0.5 regarding integrating them here.